### PR TITLE
[dev-v2.7] rancher-logging 102.0.3-rc1+up3.17.10 update

### PR DESCRIPTION
**Pull request by HarrisonWAffel**
## Problem

Various components of Rancher are using Windows images which contain Foreign Layer references. This makes mirroring these images more difficult and is a common source of user confusion when configuring windows nodes in air-gapped environments. Other PRs to rancher/rancher and rancher-monitoring have already resulted in many of these images being fixed, however logging has not yet been addressed. 

## Solution
+ Bump rancher/fluent-bit to `v1.8.10`
    + `v1.8.10` is simply a rebuilt version of `1.8.9` which no longer contains foreign layer references. There is no functional difference between the two.

## Testing
I've confirmed the chart can be installed and work as expected. I've confirmed the image does not contain foreign layers.

### CODE REVIEW

**Review by joshmeranda — APPROVED** [review]

File: regsync.yaml
Comment (lucasmlp) [inline regsync.yaml:1]: This file should not be changed by contributors. We have a GitHub action that automatically detects new docker images and updates `regsync.yaml`

**Review by lucasmlp — CHANGES_REQUESTED** [review]
Please, make changes to only one main chart per PR. Otherwise, if things goes south and we need to revert this PR, it becomes quite tricky.

File: regsync.yaml
Comment (HarrisonWAffel) [reply regsync.yaml:1]: Ah okay, good to know

Comment (HarrisonWAffel) [reply regsync.yaml:1]: I've removed the changes to `regsync.yaml` and taken out the monitoring chart changes, this PR is now only focused on the logging chart changes

**Review by lucasmlp — APPROVED** [review]
LGTM 🚀

**Pull request merged**